docs/versions/v26.4.5/guide/getting-started/canvas-guide.md

```
@@ -0,0 +1,197 @@
+search: false
+---
+
+# Using the Canvas Designer
+
+## Overview
+
+The Canvas designer (https://app.pipelineframework.org) provides a visual interface for creating and configuring pipeline applications. This document explains how to use it effectively.
+
+## Getting Started
+
+### 1. Access the Canvas
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)